AlgebraInLean/Chapter02/Sheet01.lean

```diff
@@ -3,252 +3,90 @@ import AlgebraInLean.Chapter02.Sheet00
 namespace AlgebraInLean
 
 /-
+Given a group G and a group H, a group homomorphism is a map φ from G to H which "preserves" or
+"respects" the group structure. That is, given elements a, b ∈ G, φ(a ⬝ b) = φ(a) ⬝ φ(b).
 
-# Morphisms
+In other words, you can combine a and b in G, and then apply φ, or apply φ to a and b individually,
+and then combine them in H.
 
-Morphisms are structure-preserving maps between objects in a category. Think of a category as a
```

Comment:
Confused about these deleted sections? Did you rework them onto another sheet?

Reply:
If you mean the section headings (like `# Morphisms`) I removed those since they aren't used elsewhere in the other chapters. I reworded this explanation to be more concise and put less emphasis on [abstract nonsense](https://en.wikipedia.org/wiki/Abstract_nonsense) (category theory) since I think that distracts too much from the content at hand (students that are interested can still open the link and read an explanation that's much better than anything we could write here). Reading this again, I do think that I should add back the examples of linear transformations and such.